### PR TITLE
Add JWT authentication

### PR DESCRIPTION
**Pull request by Krout0n**
# Summary
JWT authentication is implemented, the key for encoding / decoding and some claim-values are embedded at the environment variables.

- Related to https://github.com/acompany-develop/QuickMPC/pull/54

# Purpose

This PR resolves an internal-organization ticket.

# Contents

- Add authentication using JWT
- Add JWT generator, and sample input/output.

# Testing Methods Performed

- Run demo with https://github.com/acompany-develop/QuickMPC/pull/54

### CODE REVIEW

File: src/BeaverTripleService/Server/EngineToBts/E2b.go [hunk 1]
```diff
@@ -68,6 +74,84 @@ func (s *server) GetTriples(ctx context.Context, in *pb.GetTriplesRequest) (*pb.
...
+	// TODO: AuthJWT should return claim info then add it to context
+	err = AuthJWT(tokenString)
+	if err != nil {
+		return nil, err
+	}
+
+	return ctx, nil
+}
+
+func AuthJWT(tokenString string) error {
```
Comment (mdonaka) [inline src/BeaverTripleService/Server/EngineToBts/E2b.go]: Is `AuthJWT` called by other modules?
If not called, I want to change to `authJWT`.

**Review by mdonaka — COMMENTED** [review]
It would be great if you could add an env file for development and testing.
I want to eliminate the hassle of each person having to set up an env.

Image of cc
- [.env](https://github.com/acompany-develop/QuickMPC/blob/main/Config/ComputationContainer/Compute1/.env)
- [docker-compose.yml](https://github.com/acompany-develop/QuickMPC/blob/d6063355ae6f2ea6c5a35e61451f69af77ef90fe/Test/docker-compose.yml#L17-L18)

**Comment by arukuka** [comment]
> It would be great if you could add an env file for development and testing.
I want to eliminate the hassle of each person having to set up an env.

Noted with thanks.  I will add

- tools for generating jwt token
- `sample.env` file
- read it in `docker-compose.yml` for `src` build stage

File: src/BeaverTripleService/Server/EngineToBts/E2b.go (hunk 1 above)
Comment (arukuka) [reply src/BeaverTripleService/Server/EngineToBts/E2b.go]: @mdonaka Noted with thanks.  
I fixed it.

**Comment by arukuka** [comment]
@mdonaka  

I have completed the items listed below.

> > It would be great if you could add an env file for development and testing.
> > I want to eliminate the hassle of each person having to set up an env.
> 
> Noted with thanks. I will add
> 
> * tools for generating jwt token
> * `sample.env` file
> * read it in `docker-compose.yml` for `src` build stage

**Comment by mdonaka** [comment]
@arukuka 
Thank you! It's so great!
Could you please add it to https://github.com/acompany-develop/QuickMPC-BTS/blob/develop/Test/docker-compose.yml as well?

**Comment by arukuka** [comment]
@mdonaka  
:bow:
I added description in this PR.

**Review by mdonaka — APPROVED** [review]
@arukuka 
Sorry, I was misunderstanding.
No problem. Approve!

**Pull request merged**